commons/auth-filters/authn-filter/jaspi-modules/jwt-session-module/src/test/java/org/forgerock/jaspi/modules/session/jwt/ServletJwtSessionModuleTest.java

```diff
@@ -494,7 +494,7 @@ public void shouldValidateRequestWhenJwtSessionCookiePresentAndValidCoolOffPerio
         Date issuedAtTime = calendar.getTime();
 
         given(messageInfo.getRequestMessage()).willReturn(request);
-        given(messageInfo.getResponseMessage()).willReturn(response);
+        //given(messageInfo.getResponseMessage()).willReturn(response);
```

Comment:
Why this line in the test was commented?

Reply:
don't using in test